web/beacon-app/src/features/members/hooks/useUpdateProfile.tsx

```diff
@@ -16,6 +16,8 @@ export function useUpdateProfile(): ProfileUpdateMutation {
       queryClient.invalidateQueries({ queryKey: [RQK.MEMBER_LIST] });
       queryClient.invalidateQueries({ queryKey: [RQK.MEMBER_DETAIL] });
       queryClient.invalidateQueries({ queryKey: [RQK.PROFILE] });
+      queryClient.invalidateQueries({ queryKey: [RQK.ORG_DETAIL] });
```

Comment:
This change informs React Query that the data for these queries is now stale, due to a user completing the onboarding process, and should be re-fetched.

Reply:
Thanks for noticing the issue and making the fix!